tests/test_psf.py

```diff
@@ -255,5 +255,99 @@ def test_r0_500(self):
+        )
+        sed900 = galsim.SED(
+            galsim.LookupTable(
+                [200,899,900,901,1100], [0,0,1,0,0], interpolant='linear'
+            ),
+            'nm',
+            'fphotons'
+        )
+
+        bandpass = galsim.Bandpass(
+            galsim.LookupTable(
+                [200,201,1099,1100],
+                [0,1,1,0],
+                interpolant='linear'
+            ),
+            wave_type='nm'
+        )
+
+        template = {
+            'psf': {
+                'type': 'AtmosphericPSF'
+            },
+            'input': {
+                'atm_psf': {
+                    'airmass': 1.0,
+                    'rawSeeing': 1.0,
+                    'band':  'r',
+                    'screen_size': 51.2,
+                    'boresight': {
+                        'type': 'RADec',
+                        'ra': { 'type': 'Degrees', 'theta': 0.0},
+                        'dec': { 'type': 'Degrees', 'theta': 0.0}
+                    },
+                    'exponent': None,
+                    'exptime': 600.0,  # Lots of mixing.
+                    'doOpt': False,
+                    '_no2k': True,  # turn off second-kick since it's achromatic
+                }
+            },
+            'image_pos': galsim.PositionD(0,0),  # This would get set appropriately during
+                                                 # normal config processing.
+            'image' : {
+                'random_seed': 1234,
+                'draw_method' : 'phot',
+                'wcs': {
+                    'type' : 'Tan',
+                    'dudx' : 0.2,
+                    'dudy' : 0.,
+                    'dvdx' : 0.,
+                    'dvdy' : 0.2,
+                    'ra' : '@input.atm_psf.boresight.ra',
+                    'dec' : '@input.atm_psf.boresight.dec',
+                },
+                'bandpass': bandpass,
+                'size': 31
+            },
+            'gal': {
+                'type': 'DeltaFunction',
+                'flux': 1e5,
+                'sed': None
+            }
+        }
+
+        for exponent in [-0.2, -0.3]:
+            config400 = copy.deepcopy(template)
+            config400['input']['atm_psf']['exponent'] = exponent
+            config400['gal']['sed'] = sed400
+            img400 = galsim.config.BuildImage(config400)
+            sigma400 = galsim.hsm.FindAdaptiveMom(img400).moments_sigma
+
+            config900 = copy.deepcopy(template)
+            config900['input']['atm_psf']['exponent'] = exponent
+            config900['gal']['sed'] = sed900
+            img900 = galsim.config.BuildImage(config900)
+            img900.wcs = galsim.PixelScale(0.05) # just lie a little bit
+            sigma900 = galsim.hsm.FindAdaptiveMom(img900).moments_sigma
+
+            np.testing.assert_allclose(
+                sigma400/sigma900,
+                (400/900)**exponent,
+                rtol=0.02  # 2% error on value of around ~1.2 to 1.3
```

Comment:
I think I would have expected somewhat better than 2% here.  Does it improve with more photons?  I.e. is this just shot noise or a sign of something else going on?

Reply:
At the very least, the pixel convolution is there.  I did try going to more photons, didn't seem to help much.  I can try shrinking the pixel size some more...  I'm not sure what kind of systematic uncertainties to expect from hsm too.

Reply:
Ah, the pixel convolution would definitely impact this.  That's probably the main thing.

Reply:
Okay.  Got it down to 0.1% using 4x smaller pixels.

Reply:
Great.  I'm satisfied that that's the reason it's not perfect then.